### PR TITLE
Fix JavaScript build for `dev` and `demo` commands.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: lib/tasks/build-js.js [hunk 1]
```diff
@@ -39,7 +39,9 @@ module.exports = async function (cfg) {
 
 		const swcArguments = ['--filename', '-', '--no-swcrc', '-C', 'minify', '--source-maps', 'inline'];
 
-		const swc = run('swc', swcArguments, dontOutputToTerminal);
+		const swc = run('swc', swcArguments, Object.assign({
+			localDir: path.resolve(__dirname, '../../node_modules/.bin')
```
Comment (JakeChampion) [inline lib/tasks/build-js.js:43]: This shouldn't be needed :/ we use execa and execa adds the node_modules/.bin to the path I thought?

Comment (notlee) [reply lib/tasks/build-js.js:43]: without digging I can only add ¯\_(ツ)_/¯ ...works now, didn't before

Comment (notlee) [reply lib/tasks/build-js.js:43]: I'm going to write an integration test for the demo command

Comment (JakeChampion) [reply lib/tasks/build-js.js:43]: I don't know if that will work for globally installed obt though, is there a way to test?

Comment (notlee) [reply lib/tasks/build-js.js:43]: we're using the same pattern already for bower
		localDir: path.resolve(__dirname, '../../node_modules/.bin')

Comment (notlee) [reply lib/tasks/build-js.js:43]: https://github.com/sindresorhus/execa/issues/169

Comment (notlee) [reply lib/tasks/build-js.js:43]: tests over here https://github.com/Financial-Times/origami-build-tools/pulls

**Pull request merged**